src/python/zquantum/core/wip/circuit/_circuit.py

```diff
@@ -95,44 +72,40 @@ def evaluate(self, symbols_map: Dict["sympy.Symbol", Any]):
         evaluated_circuit = circuit_class(gates=evaluated_gate_list)
         return evaluated_circuit
 
-    def to_dict(self, serializable: bool = True):
-        """ Creates a dictionary representing a circuit.
+    def to_dict(self):
```

Comment:
What was the main motivation behind using `serializable` arg?
@max-radin @simonwa7 ?
Was it the fact that an issue with numpy arrays, sympy symbols or sth else?

Reply:
I was recently chatting with @alexjuda about this and I couldn't really remember why we chose to use it in the first place. I can't think of any time you would want a `dict` representation of a `Circuit` or a `Gate` without wanting it to be serializable.

Reply:
Sounds good, thanks @simonwa7  👍